src/options.cpp

```diff
@@ -611,9 +611,9 @@ void register_options(void)
    unc_add_option("sp_brace_typedef", UO_sp_brace_typedef, AT_IARF,
                   "Add or remove space between '}' and the name of a typedef on the same line.");
    unc_add_option("sp_catch_brace", UO_sp_catch_brace, AT_IARF,
-                  "Add or remove space between 'catch' and '{' if on the same line.");
+                  "Add or remove space between 'catch' and '{' in 'catch (something) { }' if on the same line.");
```

Comment:
I'm confused, how is a space added between `catch` and `{` if there is *already* non-space text between those? I guess the description should be revised to clarify?

If it's before the brace, I recommend "Add or remove space before '{' in 'catch (...) { }' if on the same line.".